core/src/main/java/org/testcontainers/images/builder/ImageFromDockerfile.java

```diff
@@ -142,7 +154,9 @@ public void onNext(BuildResponseItem item) {
 
     protected void configure(BuildImageCmd buildImageCmd) {
         buildImageCmd.withTag(this.getDockerImageName());
+        // always use withDockerfile because it honors .dockerignores and withDockerfilePath does not
```

Comment:
```suggestion
```
I think this comment isn't necessary here any more.
Does it make sense that we would set both `withDockerfilePath` and `withDockerfile` on `buildImageCmd` if both are set in `ImageFromDockerfile`?

Shouldn't one override the other?

Reply:
ultimately one will override the other, but we'll let the Docker Java API decide which one takes precedence in the event the user has set both of them. Ideally we could redirect the now-deprecated `withDockerfilePath` method to `withDockerfile` in our layer, but that may be a breaking change for some users so we can't do it